Library/Homebrew/cask/cask_loader.rb

```diff
@@ -319,15 +299,21 @@ def load(config:)
             # convert generic string replacements into actual ones
             artifact = cask.loader.from_h_hash_gsubs(artifact, appdir)
             key = artifact.keys.first
-            send(key, *artifact[key])
+            if artifact[key].nil?
```

Comment:
Maybe it'd be better to be explicit here but at the moment the only artifacts with null values are `*flight` blocks.